lib/google/gapic/templates/_client.erb

```diff
@@ -0,0 +1,2 @@
+class <%= @service.name %>Client
```

Comment:
Let's start commenting these as well (particularly the utility templates). I think it's okay to skip for this PR since they are so simple right now, but let's not develop a bad habit.

Reply:
Agreed.

Reply:
I made comments in `_modules.erb`